docs/working-with-projects/working-with-images/preheat-images.md

```diff
@@ -0,0 +1,121 @@
+1. Set the repository filter(required) by following the [doublestar pattern](https://github.com/bmatcuk/doublestar#patterns).
+1. Set the tags filter(required) by following the same [doublestar pattern](https://github.com/bmatcuk/doublestar#patterns).
+1. Set the labels filter to confirm only images with the labels can be put into the candidate list. Use comma to split 
+multiple labels, e.g.: `label1`,`label2`,`labeln` and the relationship among multiple labels are **AND**.
+1. Under certain conditions(deployment security is configured), more criteria may be visible in the policy.
+    ![extra criteria](../../../img/p2p-preheat/more-criteria-in-policy.png)
+    - If `Notary` server is installed ,and **Deployment Security** configuration option **Enable content trust** is, 
+    **Only signed images** will also be available as a criteria for the preheating policy. This means that only images 
+    with valid signatures will be preheated.
+    - If vulnerability scanner is configured ,and the **Deployment Security** configuration option 
+**Prevent vulnerable images from pulling** is set, **No vulnerability severity of [severity] and above** 
+will also be available as a preheating criteria. With this criteria, only the images whose vulnerability severity match the 
+criteria can be taken into account.
+    
+   {{< note >}}
+   The settings of these two extra criteria are directly inherited from corresponding settings of the project configuration. 
+   They cannot be changed in the preheat policy, and the only way to change them is via project configuration. If 
+   they're configured, they will be visible in the preheat policy and taken into account when calculating the 
+   phreating candidates, otherwise, they will be hidden and no influences to the policy.
+   {{< /note >}}
+
+1. For the policy **trigger**, there are multiple ways supported, choose the proper one based on your use case.
+    - **Manual**: manually start the preheating process.
+    - **Schedule**: set CRON style schedule to periodically start the preheating process.
+      * some pre-defined cron schedule patterns are provided: `Hourly`,`Daily` and `Weekly`.
+      * customize your own cron schedule by following the [cron guide](https://en.wikipedia.org/wiki/Cron)
+        - e.g.: `*/15 0 * * *`, execute policy every 15 minutes at every midnight
+    - **Event-based**: check if the image should be preheated when the related events occurred, the events includes:
+      * **OnPush**: when the image has been pushed to Harbor
+      * **OnScanComplete**: when the image has been scanned successfully (no action when scan failed)
+      * **OnLabel**: when the image has been marked with labels (no action when unmark labels)
+    
+   {{< note >}}
+   Events occurred does not mean the preheating process will be started immediately, just an evaluation process will be launched.
```

Comment:
Suggestion:

"When an event occurs, the preheating process is not started immediately. Instead an evaluation process is launched. The evaluation process will traverse the existing event-based preheat policies under the project where the target image bound in the event is pushed. If the target image matches the pre-defined filters and criteria of some event-based preheat policies, then the matched event-based preheat policies with fixed source image will be executed to complete the preheating process."